README.md

```diff
@@ -319,27 +327,38 @@ If you found a bug or want to suggest a new feature, please file a Github issue.
 
 ## Changelog
 
+- 2022-08-03 `v3.0.0`
+    - this version is not fully backward compatible because of differences in migration commands
+    - added sync of customers who made an order without account
+    - enhanced CLI: removing three `migrate-...` commands and replace them with one `migrate` with several options
+    - added migration of metadata: as metadata from commercetools side are considered `custom fields` in case of `orders` and `customers` and `attributes` in case of `products`
+    - fixed problem with deleting other api extensions while api extension registration
```

Comment:
`fixed problem with deleting other api extensions while api extension registration` -> that was fixed in previous version